.github/workflows/score_history.yml

```diff
@@ -69,17 +72,37 @@ jobs:
       - name: Install dependencies
         run: pip install numpy pandas requests python-dotenv GitPython sqlalchemy psycopg2
 
+
+      - name: Add scalingo to know hosts
+        run: echo "KNOWN_HOSTS=$(ssh-keyscan -H ssh.osc-fr1.scalingo.com)" >> $GITHUB_ENV
+
+      - name: Install SSH key
+        uses: shimataro/ssh-key-action@v2
+        with:
+          key: ${{ secrets.SCALINGO_SSH_KEY }}
+          name: id_rsa # optional
+          known_hosts: ${{ env.KNOWN_HOSTS }}
+          config: ${{ secrets.CONFIG }} # ssh_config; optional
+          if_key_exists: fail # replace / ignore / fail; optional (defaults to fail)
+
       - name: Score History
         env:
           GITHUB_REF_NAME: ${{ github.ref_name }}
           LAST_COMMIT_HASH: ${{ github.sha }}
-          SCALINGO_POSTGRESQL_SCORE_URL: ${{ secrets.SCALINGO_POSTGRESQL_SCORE_URL }}
+          SCALINGO_POSTGRESQL_SCORE_URL: ${{ secrets.SCALINGO_POSTGRESQL_TUNNEL_SCORE_URL }}
           DJANGO_BYPASS_AUTH: True
           NODE_ENV: "test"
+          SCALINGO_REGION: ${{ secrets.SCALINGO_REGION }}
+          SCALINGO_APP: ecobalyse
+          SCALINGO_API_TOKEN: ${{ secrets.SCALINGO_API_TOKEN }}
```

Comment:
Same question

Reply:
`SCALINGO_REGION` is an env variable needed by the scalingo cli tool https://doc.scalingo.com/platform/internals/regions. This value could have been public and don't really need a secret for it.
` SCALINGO_API_TOKEN` has been generated on scalingo using my user. We should  replace it with ah API key created by the "dedicated scalingo user" that we will create.